scripts/deploy_local.sh

```diff
@@ -249,15 +270,11 @@ write_envs_config() {
   rm ${SOURCE_DIR}/gs_env || true
   if [[ "${PLATFORM}" == *"Darwin"* ]]; then
     {
-      echo "export CC=/usr/local/opt/llvm@${LLVM_VERSION}/bin/clang"
-      echo "export CXX=/usr/local/opt/llvm@${LLVM_VERSION}/bin/clang++"
-      echo "export LDFLAGS=-L/usr/local/opt/llvm@${LLVM_VERSION}/lib"
-      echo "export CPPFLAGS=-I/usr/local/opt/llvm@${LLVM_VERSION}/include"
       echo "export OPENSSL_ROOT_DIR=/usr/local/opt/openssl"
       echo "export OPENSSL_LIBRARIES=/usr/local/opt/openssl/lib"
       echo "export OPENSSL_SSL_LIBRARY=/usr/local/opt/openssl/lib/libssl.dylib"
       echo "export JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home"
```

Comment:
You cannot hard-code this....

Reply:
add anFIXME: support latest java version